torchgeo/datamodules/ssl4eo_benchmark.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""SSL4EO datamodule."""
+
+from typing import Any
+
+import kornia.augmentation as K
+
+from ..datasets import SSL4EOLBenchmark
+from ..transforms import AugmentationSequential
+from .geo import NonGeoDataModule
+
+
+class SSL4EOLBenchmarkDataModule(NonGeoDataModule):
+    """LightningDataModule implementation for the SSL4EO-L Benchmark dataset.
+
+    .. versionadded:: 0.5
+    """
+
+    crop_size = 224
+
+    def __init__(
+        self, batch_size: int = 64, num_workers: int = 0, **kwargs: Any
+    ) -> None:
+        """Initialize a new SSL4EOLBenchmarkDataModule instance.
+
+        Args:
+            batch_size: Size of each mini-batch.
+            num_workers: Number of workers for parallel data loading.
+            **kwargs: Additional keyword arguments passed to
+                :class:`~torchgeo.datasets.SSL4EOLBenchmark`.
+        """
+        super().__init__(SSL4EOLBenchmark, batch_size, num_workers, **kwargs)
+
+        self.train_aug = AugmentationSequential(
```

Comment:
If you use the same aug for all splits, you can just set `self.aug` instead. However, we might want different augs for train to improve performance.

Reply:
Yes, I wrote them out because I thought it remains to be discussed whether we have additional training augs.